android/build.gradle

```diff
@@ -10,7 +10,7 @@ android {
     buildToolsVersion rootProject.hasProperty('buildToolsVersion') ? rootProject.buildToolsVersion : DEFAULT_BUILD_TOOLS_VERSION
 
     defaultConfig {
-        minSdkVersion 16
+        minSdkVersion 21
```

Comment:
I think need add new variable `DEFAULT_MIN_SDK_VERSION` and set `minSdkVersion` like than:

```
minSdkVersion rootProject.hasProperty('minSdkVersion') ? rootProject.minSdkVersion : DEFAULT_MIN_SDK_VERSION
```
otherwise it's may sometimes  gets errors on old projects:o

Reply:
I second this approach for prospective updates instead of hardcoding it to 21/16.